packages/itwinui-css/backstop/tests/index.html

```diff
@@ -68,7 +68,10 @@
 
     <style>
       @import url("./assets/demo.css");
+      @import url("@itwin/itwinui-variables");
+      @import url("@itwin/itwinui-variables/os.css");
```

Comment:
Why do you need to import `os.css` specifically if you've already imported `@itwin/itwinui-variables`? Is it because this is a url import?

Reply:
`@itwin/itwinui-variables` refers to `index.css`

we specifically want to also import `os.css` in `index.html` because there is no javascript theme switcher in here.